datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -219,6 +219,7 @@ public class Coordinator implements ZkAdapter.ZkAdapterListener, MetricsAware {
   private final Map<String, SerdeAdmin> _serdeAdmins = new HashMap<>();
   private final Map<String, Authorizer> _authorizers = new HashMap<>();
   private volatile boolean _shutdown = false;
+  private volatile boolean _notified = false;
```

Comment:
Should we rename to coordinatorStopped or some such. Notified sounds too generic.